# SetPos service sets the position of the robot
uint16 x
uint16 y
float32 a
---
bool response